### ISSUE TITLE
`git-commit` type fails with package `git-repo-info` version 1.3.0

### ISSUE THREAD

**Issue by mblayman** [ember-cli-deploy/ember-cli-deploy-revision-data#39]
The `findRepo` method in version 1.3.0 of `git-repo-info` no longer returns a string path for the repository path. It now returns an object literal with two values. This causes a path join to raise an exception.

```
- creating revision data using `git-commit`
|
+- didFail
TypeError: Path must be a string. Received { worktreeGitDir: '/srv/myapp/.git',
  commonGitDir: '/srv/myapp/.git' }
TypeError: Path must be a string. Received { worktreeGitDir: '/srv/myapp/.git',
  commonGitDir: '/srv/myapp/.git' }
    at assertPath (path.js:7:11)
    at Object.join (path.js:1211:7)
    at findRepo (/srv/myapp/client/node_modules/git-repo-info/index.js:43:20)
    at module.exports (/srv/myapp/client/node_modules/git-repo-info/index.js:144:21)
    at CoreObject.generate (/srv/myapp/client/node_modules/ember-cli-deploy-revision-data/lib/data-generators/git-commit.js:13:16)
    at Class._getData (/srv/myapp/client/node_modules/ember-cli-deploy-revision-data/index.js:59:12)
    at Class.prepare (/srv/myapp/client/node_modules/ember-cli-deploy-revision-data/index.js:36:24)
    at Object.fn (/srv/myapp/client/node_modules/ember-cli-deploy/lib/tasks/pipeline.js:159:21)
    at Pipeline._notifyPipelinePluginHookExecution (/srv/myapp/client/node_modules/ember-cli-deploy/lib/models/pipeline.js:175:19)
|
Pipeline aborted
```

**Comment by lukemelia** [comment]
Work in progress to fix this here: https://github.com/ember-cli-deploy/ember-cli-deploy-revision-data/pull/40

**Comment by lukemelia** [comment]
We would accept a PR to lock the dependency to a working version while this fix is in progress.

**Comment by lukemelia** [comment]
Fixed in v0.3.2
